homeassistant/components/rainbird/__init__.py

```diff
@@ -17,9 +22,12 @@
 )
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import discovery
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.typing import ConfigType
 
+from .coordinator import RainbirdUpdateCoordinator
+
```

Comment:
Existing issue but `SENSOR_TYPES` and `BINARY_SENSOR_TYPES` belong in their respective platforms

Reply:
Moved to platforms.